test/appium/tests/atomic/account_management/test_profile.py

```diff
@@ -176,16 +176,12 @@ def test_need_help_section(self):
         sign_in_view.create_user()
         profile_view = sign_in_view.profile_button.click()
         profile_view.help_button.click()
-        profile_view.request_feature_button.click()
-        profile_view.find_full_text('Feature Requests')
-        profile_view.click_system_back_button()
-        profile_view.submit_bug_button.click()
-        profile_view.find_full_text('Report a problem')
-        profile_view.click_system_back_button()
-        profile_view.discard_button.click()
         base_web_view = profile_view.faq_button.click()
         base_web_view.open_in_webview()
-        profile_view.find_text_part('Questions around beta')
+        base_web_view.find_text_part('Questions around beta')
```

Comment:
@antdanchenko I updated this test. Could you review it? I'll also update https://ethstatus.testrail.net/index.php?/cases/view/5391 when this PR is merged.

Reply:
@antdanchenko btw, [the test passed](https://app.saucelabs.com/tests/7e03e36e263a4ec69238959d371ebfe9/watch?auth=336bfe347c0612b3b325cf5d85139163#45).